src/app/frontend/deploy/deploy_controller.js

```diff
@@ -115,7 +128,7 @@ export default class DeployController {
    * @export
    */
   isDeployDisabled() {
-    return this.isDeployInProgress_;
+    return this.isDeployInProgress_ || this.deployLabelService_.hasDuplicationError(this.labels);
```

Comment:
You can hook custom validators to any form: https://docs.angularjs.org/guide/forms "Custom Validation
"